### DISCUSSION TITLE
Possible to add METAR data to VRS Map?

### DISCUSSION THREAD

**Discussion by BallerG** [vradarserver/vrs#101] [Q&A]
I currently use beta 3 latest version is it possible via a plugin or to add METAR data to the map for either selected airports near me or all of them?

**Comment by rikgale** [comment]
![image](https://user-images.githubusercontent.com/66202012/226752435-d08cc4c4-bae2-4966-907f-5e1adfb51bb7.png)
 Try using my layers plugin - the airports with Airport ICAO codes have links to METAR on e6bx.com
